src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -271,7 +271,7 @@ private void updateDataSelection(IReadOnlyList<DataColumnDTO> selectedDataColumn
 
       protected override void SetDefaultColumnSettings()
       {
-         AddColumnSettings(BrowserColumns.RepositoryName).WithCaption(Captions.Chart.DataBrowser.RepositoryName).GroupIndex = 0;
+         AddColumnSettings(BrowserColumns.RepositoryName).WithCaption(Captions.Chart.DataBrowser.RepositoryName).GroupIndex = -1;
```

Comment:
`-1` means "no grouping"? And what would happen if we just would say 
`AddColumnSettings(BrowserColumns.RepositoryName).WithCaption(Captions.Chart.DataBrowser.RepositoryName);` ?
Looking at the columns below: no grouping

Reply:
hehe true, I just wanted to make extra sure. -1 actually actively removes the grouping for this column if there was one till now - specifying none leaves everything as is. Of course the user can then change the groupings in the UI.

Reply:
Ok. can you add your explanation "-1 actively removes the grouping for this column if there was one till now - specifying none leaves everything as is." as a comment?
Then fine for me :)

Reply:
done